src/components/Amalgamation/BusinessTable.vue

```diff
@@ -128,8 +194,10 @@ export default class BusinessTable extends Mixins(AmalgamationMixin) {
   }
 
   key (item: AmalgamatingBusinessIF): string {
-    if (item?.type === AmlTypes.LEAR) return item.identifier
-    if (item?.type === AmlTypes.FOREIGN) return item.corpNumber
+    // make the key depend on whether the more actions menu is shown (for reactivity)
+    let x = this.showMoreActionsMenu(item) ? 'y' : 'n'
+    if (item?.type === AmlTypes.LEAR) return `${item.identifier}-${x}`
+    if (item?.type === AmlTypes.FOREIGN) return `${item.corpNumber}-${x}`
```

Comment:
The `key` is only used internally so it doesn't matter what I set it to as long as it changed when the menu visibility changed.